src/utils/hooks/useLoadApplication.tsx

```diff
@@ -112,11 +113,13 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
       const { stageId, stage, status, statusHistoryTimeCreated } = stages[0] // Should only have one result
       setApplication({
         ...application,
-        stage: {
-          id: stageId as number,
-          name: stage as string,
-          status: status as string,
-          date: statusHistoryTimeCreated.split('T')[0],
+        current: {
+          stage: {
+            id: stageId as number,
+            name: stage as string,
+          },
+          status: status as ApplicationStatus,
+          date: statusHistoryTimeCreated.split('T')[0] as Date,
```

Comment:
Can we use the DateTime library (Luxon) to handle this rather than string manipulation? The expression will just be something like:
```date: DateTime.fromISO(statusHistoryTimeCreated)```
(I've used it in `buildQueryFilters` if you want to check the syntax)

Reply:
Actually, why are we restricting it to just a date here, as opposed to the full timestamp? If we make it a DateTime object (`DateTime.fromISO(statusHistoryTimeCreated`), then we can just use the DateTime library to display it in whatever format we want when we need it.

Reply:
Sure. I don't think was a good thing to be removing the timestamp from it as well. I'll change to what you suggested now